doc/api/webcrypto.md

```diff
@@ -1022,21 +1020,43 @@ are simple JavaScript dictionary objects.
 ### Class: `AlgorithmIdentifier`
 
 <!-- YAML
-added:
-  - v18.4.0
-  - v16.17.0
+added: v15.0.0
```

Comment:
Do you have a reference of when that was introduced?

Reply:
These are all just interfaces that we document as classes. This is called out in the doc. And they are all baseline WebCryptoAPI, hence 15.0.0 when webcrypto got introduced.